gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -104,7 +106,8 @@ public abstract class KafkaJobStatusMonitor extends HighLevelConsumer<byte[], by
   private final StateStore<org.apache.gobblin.configuration.State> stateStore;
   private final ScheduledExecutorService scheduledExecutorService;
   private static final Config RETRYER_FALLBACK_CONFIG = ConfigFactory.parseMap(ImmutableMap.of(
-      RETRY_TIME_OUT_MS, TimeUnit.HOURS.toMillis(24L), // after a day, presume non-transient and give up
+      // keeping the retry timeout less until we configure retryer to retry only the transient exceptions
+      RETRY_TIME_OUT_MS, TimeUnit.MINUTES.toMillis(30L), // after 30 minutes, presume non-transient and give up
```

Comment:
I'm not convinced that 24 hours is the wrong length, esp. as you'll now skip the `SQLIntegrityConstraintViolationException`s.  but if you really think it needs shortening, I wouldn't go so far as only 30 mins.  maybe 6 or 12 hours at least...

Reply:
ok, changing to 6 hours. how much time should a transient problem need to get resolved?